developer-docs-site/docs/guides/prover-guides/index.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Move Prover"
+slug: "move-prover"
+---
+
+# Move Prover
+
+The Move Prover supports formal specification and verification of Move code. It can automatically prove
+logical properties of Move smart contracts, while providing a user experience similar to a type checker or linter.
+It's purpose is to make contracts more *trustworthy*, specifically:
+
+- Protect massive assets managed by the Diem blockchain from smart contract bugs
```

Comment:
the Aptos blockchain instead of Diem?

Reply:
I took care of this while making my edits. Thanks for the catch, Teng!